CMakeLists.txt

```
@@ -104,6 +104,22 @@ if( HAVE_SERIALBOX )
   list(APPEND CLOUDSC_DEFINITIONS HAVE_SERIALBOX)
 endif()
 
+
+### Mix variants
+
+ecbuild_add_option( FEATURE CLOUDSC_MIX
+        DESCRIPTION "Build the mix version CLOUDSC (Fortran driver + OpenMP/OpenACC + HIP/CUDA/SYCL" DEFAULT OFF
+        CONDITION Serialbox_FOUND OR HDF5_FOUND
+)
```

This has a hard dependency on the availability of kernel language and offload model, so the option should probably look like this:

```suggestion
ecbuild_add_option( FEATURE CLOUDSC_MIX
                    DESCRIPTION "Build the mix version CLOUDSC (Fortran driver + OpenMP/OpenACC + HIP/CUDA/SYCL" 
                    DEFAULT ON
                    CONDITION (HAVE_CUDA OR HAVE_HIP OR HAVE_SYCL) AND (HAVE_ACC OR HAVE_OMP)
)
```